apps/src/code-studio/pd/application/facilitator1920/Section6Submission.jsx

```diff
@@ -1,23 +1,34 @@
 import React from 'react';
 import {FormGroup} from "react-bootstrap";
 import LabeledFormComponent from "../../form_components/LabeledFormComponent";
-import {PageLabels, SectionHeaders} from '@cdo/apps/generated/pd/facilitator1920ApplicationConstants';
+import {
+  PageLabels,
+  SectionHeaders,
+  TextFields
+} from '@cdo/apps/generated/pd/facilitator1920ApplicationConstants';
 
-export default class Section7Submission extends LabeledFormComponent {
-  static labels = PageLabels.section7Submission;
+export default class Section6Submission extends LabeledFormComponent {
+  static labels = {...PageLabels.section6Submission, ...PageLabels.section1AboutYou};
 
   static associatedFields = [
-    ...Object.keys(PageLabels.section7Submission)
+    ...Object.keys(PageLabels.section6Submission)
   ];
 
   render() {
     return (
       <FormGroup>
-        <h3>Section 7: {SectionHeaders.section7Submission}</h3>
+        <h3>Section 6: {SectionHeaders.section6Submission}</h3>
```

Comment:
Having these numbers hard-coded in multiple separate files makes me nervous. Can we add to our tech debt to find a way to organize this so we aren't dependent on manually updating the numbers?